.gitignore

```diff
@@ -7,6 +7,7 @@ cmake-out/
 # Common bazel output directories
 bazel-*
 !bazel-dependency*.cfg
```

Comment:
Do we still need the `!bazel-dependency*.cfg` line?

Reply:
Nope, removed.